build/nodejs/build-22.sh

```diff
@@ -26,6 +26,9 @@ DESC+="servers."
 
 MAJVER=${VER%%.*}
 
+# fails to build with gcc 12
+((GCCVER = 12)) && set_gccver 11
```

Comment:
do you actually want to assign ? GCCVER to 12 ? If not, use two ==

```suggestion
((GCCVER == 12)) && set_gccver 11
```

Reply:
That was definitely an oversight, thanks for catching it.